src/chrome/content/rules/Rackspace.xml

```diff
@@ -57,11 +54,11 @@
 	<target host="mycloud.rackspace.com" />
 	<target host="partners.rackspace.com" />
 	<target host="status.rackspace.com" />
+	<target host="stories.rackspace.com" />
 
 	<securecookie host=".+" name=".+" />
 
-	<rule from="^http://images\.cdn\.rackspace\.com/"
-		to="https://034d24a88b3e71fd72a6-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/" />
+	<rule from="^http://images\.cdn\.rackspace\.com/" to="https://034d24a88b3e71fd72a6-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/" />
```

Comment:
Do we have a real testcase for this?

Reply:
No.

Reply:
Okay, let's use `http://images.cdn.rackspace.com/home/openstack.svg`